src/index.html

```diff
@@ -11,7 +11,105 @@
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname">
+        </label>
+        <label class="form-field">
+          Name:
+          <input type="text" name="name">
+        </label>
+        <label class="form-field">
+          How old are you?
+          <input
+          type="number"
+          name="age"
+          min="12"
+          max="100"
+          value="12"
+        >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birthday">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement:
+          <input type="checkbox" name="agree">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input type="email" name="email"
+          required placeholder="email@example.com">
+        </label>
+        <label class="form-field">
+          Password:
+          <input type="password" name="password"
+          required minlength="8" maxlength="32">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="cats" value="y">
+            Yes
+          <input type="radio" name="cats" value="n">
+            No
+        </label>
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+        <label class="form-field">
+          What are your favorite brands of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input type="range" name="rating" value="0">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea name="comments"></textarea>
+        </label>
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+    </form>
+    <form onsubmit="onSubmit()">
+      <label>
+        <input type="submit" value="Submit">
+      </label>
+    </form>
```

Comment:
На твоей странице должен быть один тег form, этот лишний (перенеси onsubmit с него на первый тег)
```suggestion
        <input type="submit" value="Submit">
    </form>
```